contracts/src/v0.8/shared/enumerable/EnumerableMapAddresses.sol

```diff
@@ -137,4 +139,88 @@ library EnumerableMapAddresses {
   function get(AddressToBytes32Map storage map, address key) internal view returns (bytes32) {
     return map._inner.get(bytes32(uint256(uint160(key))));
   }
+
+  struct AddressToBytesMap {
```

Comment:
Let's create a new file for this and not put it in `EnumerableMapAddresses.sol`

Reply:
Don't think we need a new file, this struct is logically grouped to contain all `AddressToX` maps - same structure as OpenZeppelin's lib

Reply:
yes logical gtouping of all Address based mappings